src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.html

```diff
@@ -27,6 +27,7 @@ <h3>{{ 'create-news.tags-title' | translate }}</h3>
                 (click)="addFilters(filter)"
                 type="button"
                 [ngClass]="{ 'filters-color': filter.isActive }"
+                class="tagNews"
```

Comment:
We should use not the camel case in html classes/ids, but - sign to separate words

Reply:
done